distribution/kernel/carbon-home/dbscripts/db2.sql

```diff
@@ -960,3 +960,96 @@ BEGIN ATOMIC
        = (NEXTVAL FOR UM_GROUP_UUID_DOMAIN_MAPPER_SEQUENCE);END/
 
 CREATE INDEX UUID_GRP_UID_TID ON UM_GROUP_UUID_DOMAIN_MAPPER(UM_GROUP_ID, UM_TENANT_ID)/
+
+-- ################################
+-- ORGANIZATION MANAGEMENT TABLES
+-- ################################
+
+CREATE TABLE IF NOT EXISTS UM_ORG (
+            UM_ID VARCHAR(255) NOT NULL,
```

Comment:
Any reason why we use the size `255`? all the places we use UUID we can use `36` right?

Reply:
@vinumaddumage Let's create a separate PR to address this in all required places.